tests/index.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../index');
+
+chai.should();
+
+chai.use(chaiHttp);
+
+describe('GET /', () => {
+  it('it should get homepage', done => {
+    chai
+      .request(server)
+      .get('/')
+      .end((err, res) => {
+        res.should.have.status(200);
+        done(err);
+      });
+  });
+});
+
+describe('Invalid route', () => {
+  it('It should return invalid route', done => {
+    chai
+      .request(server)
+      .get('/auth/signup')
```

Comment:
💡 This endpoint might exist in future, where we'll be creating new staff user from admin. Maybe give it a random `path` so that we know it will not exist.